### PR TITLE
Adds icons; combines questions as requested; moves progress bar from …

### PR DESCRIPTION
**Pull request by jessicamarcus**
- adds icons
- `nameOnRecord` and `birthDate` questions have been combined
- `parentsNames` and `parentsMarried` questions have been combined
- progress bar originally reflected the question flow; it now reflects the overall birth cert request flow

### CODE REVIEW

File: services-js/registry-certs/client/birth/constants.ts [hunk 1]
```diff
@@ -19,11 +19,13 @@ export const BIRTH_BREADCRUMB_NAV_LINKS: BreadcrumbNavigation = {
   },
 };
 
-export const QUESTIONS: Question[] = [
-  'forSelf',
+export const STEPS: Step[] = [
```
Comment (fionawhim) [inline services-js/registry-certs/client/birth/constants.ts:22]: good rename

File: services-js/registry-certs/client/birth/questions/ForWhom.tsx [hunk 2]
```diff
@@ -26,28 +29,40 @@ interface State {
  * Initial question of the workflow. If the user is not requesting their
  * own record, the howRelated question is also asked.
  */
-export default class ForSelf extends React.Component<Props, State> {
+export default class ForWhom extends React.Component<Props, State> {
```
Comment (fionawhim) [inline services-js/registry-certs/client/birth/questions/ForWhom.tsx:32]: love it!

**Review by fionawhim — APPROVED** [review]
LGTM!

File: services-js/registry-certs/client/birth/questions/ForWhom.tsx (hunk 2 above)
Comment (jessicamarcus) [reply services-js/registry-certs/client/birth/questions/ForWhom.tsx:32]: 🤓

**Pull request merged**